.github/workflows/release.yaml

```diff
@@ -1,23 +1,25 @@
 name: Release GoiSCSI
 # Invocable as a reusable workflow
 # Can be manually triggered
-on:  # yamllint disable-line rule:truthy
+on: # yamllint disable-line rule:truthy
```

Comment:
Could you revert this change to fix lint warning?

Reply:
Updated in recent commit.